index.html

```diff
@@ -0,0 +1,30 @@
+<!doctype html>
+<html lang="ru">
+<head>
+	<meta charset="UTF-8">
+	<title>Document</title>
+	<link rel="stylesheet" href="style.css" type="text/css" />
+</head>
+<body>
+	
+	<div id="container">
+		<div id="start-container">
+			<div class="title">Для сортировки введите числа разделяя их запятой.</div>
+			<form class="form">
+			<input type="text" name="form" id="sort" class="input" placeholder="например: 1,3,10,884,3,4"/>
```

Comment:
I know. But I like this method, so that it is the fastest and the least you need to write code. In the future I will write as you speak. To teach myself a more correct style,but though less elegant.